neo_force_scheme/neo_force_scheme.py

```diff
@@ -198,6 +212,7 @@ def transform(
             n_dimension: Optional[int] = 2,
             fix_column_to_z_projection_axis=None,
             z_axis_moving_range: Optional[Tuple[float, float]] = (0, 0),
+            confidence_interval: Optional[float] = 0.99,
```

Comment:
I noticed that sometimes the default confidence_interval is 0.99 and sometimes (elsewhere in the code) its 1.0. Is there a reason for that?

Reply:
The default should be 1.0, while 0,99 should only be the default of get_z_score() because we will never use it if confidence_interval is 1.0. Here in line 215 is a typo. sorry for the inconvenience.

Reply:
Also, I'd add in the :params comment what are good values for condifence_interval. Such as 1.0 if (bla) and 0.9 if (something else). Can you add that too please?

Reply:
Comment added at neo_force_scheme.py line 330 - 340